From 74e09c3a95961163e97a206d6847bec9dcabab6b Mon Sep 17 00:00:00 2001
From: Egbert Eich <eich@freedesktop.org>
Date: Tue, 4 Feb 2014 14:13:53 +0100
Subject: Back out cf93beb - restore numeric keypad separator for FR/IT/NL

This reverts commit cf93beb47f9907946dd72ca4d481ffbbbadda53a

  Let <KPDL> key on it,fr and nl keyboards generate KP_Separator.

  Notwithstanding the never ending discussion what character this
  key on the numeric keypad should produce this key should produce
  what is imprinted on the keyboards. After all all three countries
  are observing the SI standards.
  This makes this layout more consistent and match for instance
  what is done in the de layout.

It turned out that the information about the keypad separator
for the fr, it and nl keyboard layouts were obtained from what
glibc sets as delimiter for the respective language.
This however doesn't coincide what is imprinted on the keyboards
for the respective languages.

Signed-off-by: Egbert Eich <eich@freedesktop.org>

diff --git a/symbols/fr b/symbols/fr
index 9c89db8..3898b1c 100644
--- a/symbols/fr
+++ b/symbols/fr
@@ -36,8 +36,6 @@ xkb_symbols "basic" {
     key <AB09>	{ [     colon,      slash, periodcentered,   division ]	};
     key <AB10>	{ [    exclam,    section, dead_belowdot, dead_abovedot ] };
 
-    include "kpdl(comma)"
-
     include "level3(ralt_switch)"
 };
 
diff --git a/symbols/it b/symbols/it
index 8a66a30..2673cf9 100644
--- a/symbols/it
+++ b/symbols/it
@@ -33,8 +33,6 @@ xkb_symbols "basic" {
 
     key <LSGT>	{ [      less,    greater,guillemotleft,guillemotright] };
 
-    include "kpdl(comma)"
-
     include "level3(ralt_switch)"
 };
 
diff --git a/symbols/nl b/symbols/nl
index 1dde405..76cb018 100644
--- a/symbols/nl
+++ b/symbols/nl
@@ -49,8 +49,6 @@ xkb_symbols "basic" {
 
     key <LSGT>	{ [bracketright, bracketleft,       bar,    brokenbar ]	};
 
-    include "kpdl(comma)"
-
     include "level3(ralt_switch)"
 };
 
-- 
cgit v0.10.2

